flow/cmd/logged_interceptor.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"context"
+
+	"go.temporal.io/sdk/interceptor"
+	"go.temporal.io/sdk/workflow"
+)
+
+type LoggedWorkflowInboundInterceptor struct {
+	interceptor.WorkflowInboundInterceptorBase
+	Next interceptor.WorkflowInboundInterceptor
```

Comment:
```suggestion
```
Why not get it from Base?

Reply:
This was just experimental and I was testing